common/pipeline_storage/src/main/scala/uk/ac/wellcome/pipeline_storage/PipelineStorageStream.scala

```diff
@@ -95,3 +77,72 @@ class PipelineStorageStream[In, Out, MsgDestination](
       }
     )
 }
+
+object PipelineStorageStream extends Logging {
+
+  def batchRetrieveFlow[T](config: PipelineStorageConfig,
```

Comment:
Where is this being used?

Reply:
Nowhere yet, I am going to use it in my next PR